### PR TITLE
Inject content cleanup

### PR DESCRIPTION
**Pull request by davidrapson**
Following on from https://github.com/biglotteryfund/blf-alpha/pull/2559 inlines a bunch of single use inject* methods to simplify router logic

### CODE REVIEW

**Comment by davidrapson** [comment]
@mattandrews Not _quite_ finished, need to fix up a few things so might ask you to review again shortly

**Comment by davidrapson** [comment]
@mattandrews Mind giving this another look?

File: common/inject-content.js [hunk 1]
```diff
@@ -165,120 +164,12 @@ async function injectFlexibleContent(req, res, next) {
...
-
 module.exports = {
     injectBreadcrumbs,
     injectCopy,
     injectFlexibleContent,
-    injectFundingProgramme,
     injectHeroImage,
     injectListingContent,
-    injectMerchandise,
-    injectOurPeople,
```
Comment (mattandrews) [inline common/inject-content.js]: 👋 to the creepiest-named method in our entire codebase

Comment (davidrapson) [reply common/inject-content.js]: 💉 👨 💉  💉 👩‍⚕ 💉

**Pull request merged**